BSP/Simulator/Data/critical_overtemp.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
I apologize for the confusion. We will need an overtemperature test for charging and discharging. This means a total of 10 trials. For charging, have a temperature higher than 45000 and a negative current. I will update this in the one note as well.

Reply:
k will do